src/problems/richardsmitchellettesudoku.clj

```diff
@@ -0,0 +1,58 @@
+(ns problems.richardsmitchellettesudoku)
+
+(defrecord State [board])
+
+(defn getSection [board box] (nth board (int (/ box 3))))
```

Comment:
What exactly are a "section" and a "box", and what's the difference between them? That made reading a lot of this confusing.